<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="controll_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_widget">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>541</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Interface type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="inter_combo">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>--Choose one--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wifi(server)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Connection status:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="interface_status_label">
             <property name="text">
              <string>Unavalible</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,2">
         <item>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Controll type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="controll_combo">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <item>
            <property name="text">
             <string>--Choose one--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gamepad</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Keyboard</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Connection status:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="controll_status_label">
             <property name="text">
              <string>Unavalible</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Controller information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="contr_info">
           <property name="text">
            <string>No data avalible</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="param_tab">
     <attribute name="title">
      <string>Parameters</string>
     </attribute>
     <widget class="QTableWidget" name="param_table">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>273</width>
        <height>237</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <row>
       <property name="text">
        <string>czujnik 1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>czujnik 7</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Detected</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>controll_tab</tabstop>
  <tabstop>inter_combo</tabstop>
  <tabstop>controll_combo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
